perfkitbenchmarker/spark_service.py

```diff
@@ -151,6 +151,11 @@ def GetMetadata(self):
       basic_data.update({'num_workers': str(self.spec.worker_group.vm_count),
                          'worker_machine_type':
                          str(self.spec.worker_group.vm_spec.machine_type)})
+
+    if self.spec.worker_group.vm_spec.num_local_ssds is not None and \
```

Comment:
@ehankland Now, dataproc overrides the getmetadata method and evaluates the ssd attributes. Will have to look into how this is handled for PKB managed clusters in a later PR. Let me know if this is better